src/endpoint/endpoint.js

```diff
@@ -294,6 +295,11 @@ function create_endpoint_handler(server_type, init_request_sdk, { virtual_hosts,
                 return fork_count_handler(req, res);
             } else if (req.url.startsWith('/endpoint_fork_id')) {
                 return endpoint_fork_id_handler(req, res);
+            } else if (req.url.startsWith('/_')) {
+                // internals non S3 requests
+                if (req.url === '/_version') {
+                    return version_handler(req, res);
+                }
```

Comment:
How about this - I think it is easier to add more "internal api" calls under a common internal path prefix such as `/_/<api>` rather than think of each of those api names as starting with an underscore.
Also need to handle the case of unrecognized api call.
```suggestion
            } else if (req.url.startsWith('/_/')) {
                // internals non S3 requests
                const api = req.url.slice('/_/'.length);
                if (api === 'version') {
                    return version_handler(req, res);
                } else {
                    return internal_api_error(req, res, `Unknown API call ${api}`);
                }
```